src/bootstrap/download-ci-llvm-stamp

```diff
@@ -1,13 +1,4 @@
 Change this file to make users of the `download-ci-llvm` configuration download
 a new version of LLVM from CI, even if the LLVM submodule hasn’t changed.
 
-<<<<<<< HEAD
-Last change is for: https://github.com/rust-lang/rust/pull/126298
-
-
-(white space intentional to avoid merge conflicts)
-
-Last Ferrocene change is for: https://github.com/ferrocene/ferrocene/pull/731
```

Comment:
is this not used by us for anything?

Reply:
looks like it's only needed when we want to force people to download a fresh llvm, which we rarely need separately of upstream applying the same force

it also constantly conflicts, which invalidates the comment